src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -337,3 +397,15 @@ def _set_prometheus_targets(prometheus_url: str):
             'url': prometheus_url.replace('http://', '').replace('https://', '')
         }
         mgr.mon_command(cmd)
+
+def _remove_prometheus_targets(prometheus_url: str):
+    orch_backend = mgr.get_module_option_ex('orchestrator', 'orchestrator')
+    try:
+        if orch_backend == 'cephadm':
+            cmd = {
+                'prefix': 'orch prometheus remove-target',
+                'url': prometheus_url.replace('http://', '').replace('https://', '')  # noqa E501 #pylint: disable=line-too-long
+            }
+            mgr.mon_command(cmd)
+    except KeyError:
+        pass
```

Comment:
should't we at least log the error message in case of `keyError` ?

Reply:
Done